yosh/shell.py

```diff
@@ -54,14 +53,16 @@ def shell_loop():
         # Make it more looks like bash command prompt
         if os.getcwd() == os.getenv('HOME'):
             dir = "~"
+        elif os.getcwd() == os.getenv('HOME') + "/yosh":
+            dir = ""
```

Comment:
This case is not necessary. When we are in `/home/user/yosh`, it should display `yosh $` to reflect what directory that we are in.